source/agora/node/Registry.d

```diff
@@ -359,14 +370,24 @@ public class NameRegistry: NameRegistryAPI
 
     ***************************************************************************/
 
-    private void doAXFR (in ZoneData zone, ref Message reply) @safe
+    private void doAXFR (string TemplateRegistry = "") (ZoneData!(TemplateRegistry) zone, ref Message reply) @safe
     {
         log.info("Performing AXFR for {} ({} entries)",
-                 zone.root.value, zone.map.length);
+                 zone.root.value, zone.count());
```

Comment:
If you go with `opApply`, you might want to move this message in two:
`Performing AXFR for {}`
-> Add `+1` to a count at each iteration
`AXFR for {} done: {} entries`

Or just remove the count completely 🤷